poetry/core/factory.py

```diff
@@ -134,6 +159,154 @@ def create_poetry(self, cwd=None):  # type: (Optional[Path]) -> Poetry
 
         return Poetry(poetry_file, local_config, package)
 
+    def create_dependency(
```

Comment:
I will change this one to a class method. I'll leave the `create_poetry()` one as is for now to avoid unnecessary clutter to this PR.

We can always do this in a separate PR.

Reply:
Sounds reasonable.